### PR TITLE
ci: update labeler

### PR DESCRIPTION
**Pull request by teolemon**
### What
- ci: update labeler

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+documentation:
+- CONTRIBUTING.md
+- LICENSE
+- README.md
+- backend/README.md
+- doc/how-to-guides/develop-with-neo4j.md
+- doc/how-to-guides/docker-compose-setup.md
+- doc/introduction/setup-dev.md
+- doc/references/taxonomy-parser.md
+- doc/assets/.empty
```
Comment (aadarsh-ram) [inline .github/labeler.yml:16]:
```suggestion
- CONTRIBUTING.md
- LICENSE
- README.md
- backend/README.md
- doc/**/*
```

File: .github/labeler.yml [hunk 2]
```diff
@@ -1,5 +1,74 @@
...
+
+docker:
+- backend/Dockerfile
+- docker-compose.yml
+- docker/dev.yml
```
Comment (aadarsh-ram) [inline .github/labeler.yml:21]:
```suggestion
- backend/Dockerfile
- taxonomy-editor-frontend/Dockerfile
- docker-compose.yml
- docker/dev.yml
```

File: .github/labeler.yml [hunk 3]
```diff
@@ -1,5 +1,74 @@
...
+- backend/editor/graph_db.py
+- backend/editor/models.py
+- backend/editor/normalizer.py
+- backend/editor/settings.py
+- backend/requirements.txt
+- backend/sample/load.py
+- backend/sample/schema.json
+- backend/sample/test-neo4j.json
+- backend/sample/test.json
+- backend/sample/test.txt
```
Comment (aadarsh-ram) [inline .github/labeler.yml:37]:
```suggestion
- backend/**/*
```

File: .github/labeler.yml [hunk 4]
```diff
@@ -1,5 +1,74 @@
...
+
+dependancies:
+- backend/requirements.txt
+- parser/requirements-dev.txt
+- parser/requirements-test.txt
+- parser/requirements.txt
```
Comment (aadarsh-ram) [inline .github/labeler.yml:43]:
```suggestion
dependencies:
- backend/requirements.txt
- parser/requirements-dev.txt
- parser/requirements-test.txt
- parser/requirements.txt
- taxonomy-editor-frontend/package.json
- taxonomy-editor-frontend/package-lock.json
```

File: .github/labeler.yml [hunk 5]
```diff
@@ -1,5 +1,74 @@
...
+
+api:
+- backend/editor/api.py
+
+nginx:
+- conf/nginx.conf
+
+neo4j:
+- neo4j/.empty
```
Comment (aadarsh-ram) [inline .github/labeler.yml:52]:
```suggestion
neo4j:
- neo4j/**/*
```

File: .github/labeler.yml [hunk 6]
```diff
@@ -1,5 +1,74 @@
...
+- parser/openfoodfacts_taxonomy_parser/unparser.py
+- parser/pyproject.toml
+- parser/requirements-dev.txt
+- parser/requirements-test.txt
+- parser/requirements.txt
+- parser/setup.cfg
+- parser/tests/__init__.py
+- parser/tests/conftest.py
+- parser/tests/data/test.txt
+- parser/tests/integration/__init__.py
```
Comment (aadarsh-ram) [inline .github/labeler.yml:68]:
```suggestion
- parser/**/*
```

File: .github/labeler.yml [hunk 7]
```diff
@@ -1,5 +1,74 @@
...
+
+tests:
+- parser/tests/__init__.py
+- parser/tests/conftest.py
+- parser/tests/data/test.txt
+- parser/tests/integration/__init__.py
```
Comment (aadarsh-ram) [inline .github/labeler.yml:74]:
```suggestion
- parser/tests/**/*
```

**Review by aadarsh-ram — CHANGES_REQUESTED** [review]
@teolemon I think we can use the `**/*` filepaths in this file, since it would automatically match the files in the subfolders (even if new files come, it'd get matched).

File: .github/labeler.yml [hunk 8]
```diff
@@ -1,5 +1,77 @@
...
+documentation:
+- CONTRIBUTING.md
+- LICENSE
+- README.md
+- backend/README.md
+- doc/how-to-guides/develop-with-neo4j.md
+- doc/how-to-guides/docker-compose-setup.md
+- doc/introduction/setup-dev.md
+- doc/references/taxonomy-parser.md
+- doc/assets/.empty
```
Comment (teolemon) [inline .github/labeler.yml:16]:
```suggestion
- doc/assets/.empty
- doc/**/*
```

File: .github/labeler.yml (hunk 1 above)
Comment (teolemon) [reply .github/labeler.yml:16]: I added - doc/**/* but I want a list of important files we can point people to ( code pointers)

File: .github/labeler.yml [hunk 9]
```diff
@@ -1,5 +1,78 @@
...
+- backend/editor/graph_db.py
+- backend/editor/models.py
+- backend/editor/normalizer.py
+- backend/editor/settings.py
+- backend/requirements.txt
+- backend/sample/load.py
+- backend/sample/schema.json
+- backend/sample/test-neo4j.json
+- backend/sample/test.json
+- backend/sample/test.txt
```
Comment (teolemon) [inline .github/labeler.yml:39]:
```suggestion
- backend/sample/test.txt
- backend/**/*
```

File: .github/labeler.yml [hunk 10]
```diff
@@ -1,5 +1,79 @@
...
+- parser/openfoodfacts_taxonomy_parser/unparser.py
+- parser/pyproject.toml
+- parser/requirements-dev.txt
+- parser/requirements-test.txt
+- parser/requirements.txt
+- parser/setup.cfg
+- parser/tests/__init__.py
+- parser/tests/conftest.py
+- parser/tests/data/test.txt
+- parser/tests/integration/__init__.py
```
Comment (teolemon) [inline .github/labeler.yml:73]:
```suggestion
- parser/tests/integration/__init__.py
- parser/**/*
```

File: .github/labeler.yml [hunk 11]
```diff
@@ -1,5 +1,80 @@
...
+- parser/**/*
+
+tests:
+- parser/tests/__init__.py
+- parser/tests/conftest.py
+- parser/tests/data/test.txt
+- parser/tests/integration/__init__.py
```
Comment (teolemon) [inline .github/labeler.yml:80]:
```suggestion
- parser/tests/integration/__init__.py
- parser/tests/**/*
```

**Pull request merged**